src/algos/time_iteration.jl

```diff
@@ -103,11 +103,11 @@ If the stochastic process for the model is not explicitly provided, the process
 * `dr`: Solved decision rule.
 """
 function time_iteration(model, process, init_dr;
-      verbose::Bool=true, maxit::Int=100, tol::Float64=1e-8, infos::Bool=false)
+      verbose::Bool=true, maxit::Int=100, tol::Float64=1e-8, details::Bool=false)
 
     # get grid for endogenous
-    gg = model.options.grid
-    grid = CartesianGrid(gg.a, gg.b, gg.orders)  # temporary compatibility
+    grid = model.grid

```

Comment:
One idea here...

Should we have `get_grid(::Model)` like we have `get_grid(::SModel`), or do you prefer accessing the grid field?

The reason I ask is that it might make things easier to do "just in time" field updating (e.g. if the calibration has changed) if we go through a method instead of accessing the field itself.